src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -18,405 +18,412 @@
 using Uno.Equality;
 using Uno.Logging;
 using Uno.UI.SourceGenerators.XamlGenerator.XamlRedirection;
+using Mono.Cecil;
```

Comment:
Is Cecil actually being used? (Or the using below)

Reply:
I don't know, it was from your commit I think :). Will check.